$MirOS: ports/emulators/qemu/snapshot/patches/patch-hw_sh_intc_c,v 1.1 2008/05/07 15:53:11 tg Exp $
--- hw/sh_intc.c.orig	Wed Sep 23 19:01:08 2009
+++ hw/sh_intc.c	Sat Oct 17 16:35:43 2009
@@ -106,6 +106,7 @@ int sh_intc_get_pending_vector(struct in
     }
 
     assert(0);
+    return (0);		/* shut up gcc */
 }
 
 #define INTC_MODE_NONE       0
